governance/src/test/java/org/apache/servicecomb/governance/GovernancePropertiesTest.java

```diff
@@ -178,24 +178,24 @@ public void test_match_properties_changed() {
   public void test_bulkhead_properties_changed() {
     dynamicValues.put("servicecomb.bulkhead.demo-bulkhead", "rules:\n"
         + "maxConcurrentCalls: 2\n"
-        + "maxWaitDuration: 2000");
+        + "maxWaitDuration: 2000ms");
```

Comment:
add new test cases, do not delete old test case for it is valid.

Reply:
done